src/Components/Popover.svelte

```diff
@@ -4,8 +4,8 @@
   const dispatch = createEventDispatcher();
 
   let once = (el, evt, cb) => {
-    function handler() {
-      cb.apply(this, arguments);
+    function handler () {
+      cb();
```

Comment:
```suggestion
    function handler (...args) {
      cb.apply(this, args);
```

Maybe this could fix your problem while maintaining the original functionality of this function.  I can't guarantee that I'll never use `arguments` again in this repository, but perhaps its a good stop-gap.